README.md

```diff
@@ -12,22 +12,37 @@
 
 </div>
 
-`torchtitan` is currently in a pre-release state and under extensive development. Currently we showcase pre-training **Llama 3.1** LLMs of various sizes from scratch. To use the latest features of `torchtitan`, we recommend using the most recent PyTorch nightly.
+`torchtitan` is currently in a pre-release state and under extensive development. Currently we showcase training Llama 3.1 LLMs at scale. We are working on other types of generative AI models, including LLMs with MoE architectures, multimodal LLMs, and diffusion models, in the [`experiments`](torchtitan/experiments) folder.
+To use the latest features of `torchtitan`, we recommend using the most recent PyTorch nightly.
+
+
+## Latest News
+- [2025/04] Our [paper]((https://arxiv.org/abs/2410.06511)) is accepted by [ICLR 2025]((https://iclr.cc/virtual/2025/poster/29620)). The poster will be presented on Friday April 25th.
```

Comment:
"has been accepted" ("is accepted" is a mismatch in tense).

Reply:
You can have a badge at the top too if you like https://github.com/zsxoff/arxiv-badge